DoD and Numbering format (#252)
finos/common-cloud-controls: docs/governance/community-policies/definition-of-done.md

> # Definition of Done (DoD) for a Service

Reviewer: Is this intended to include the end-to-end delivery of a service, both taxonomy definition and control definitions?

Author: No this was written with only the taxonomy in mind. Happy if someone can extend this to cover control definition

Service in this context is anything that can not be further categorized into subcategories. For example _Relational Databases_ is a service where _Databases_ is considered as a higher level grouping of common features among all different types of databases.

> Stated below is the set of criteria that a service must meet for the team to consider it complete and ready for its consumers.

Reviewer: ```suggestion
Stated below is the set of criteria that a service must meet to be ready for delivery.
```

> - A comprehensive set of minimum viable features should be identified for the service.

Reviewer: ```suggestion
- A comprehensive set of minimum viable features has been identified for the service.
```

changing phrasing to past tense (as in, this must be true at the time it is considered done)

> - All possible higher-level groupings of features should be identified for the service.

Reviewer: ```suggestion
- All possible higher-level groupings of features have been defined for the service.
```

> - A folder should be created with the name of the service, under **services** folder in the repository which clearly depicts its place in the taxonomy. E.g. `services > databases > relational` where _relational_ is the service name and _databases_ is the higher-level grouping

Reviewer: ```suggestion
- A folder has been created under **services** folder, with the name of the service, according to its place in the taxonomy. (e.g. `services/databases/relational/` where _relational_ is the service name and _databases_ is the higher-level grouping)
```

> - All identified features should be documented in a `taxonomy.md` file under the folder created for the service.

Reviewer: ```suggestion
- All identified have been documented in a `taxonomy.md` file within the directory created for the service.
```

> - All identified features should be numbered according to the _common-cloud-control_ guidelines.

Reviewer: ```suggestion
- All common features have been numbered according to the (numbering format)[../numbering-format.md].
```

> - A description should be provided for all identified features.

Reviewer: ```suggestion
- A description has been provided for all common features.
```

> - An asset describing the ways to test the presence of each control should be documented in feature file with the name `<service-name>.feature`.

Reviewer: ```suggestion
- An gherkin feature file describing the ways to test the presence of each control has been cretated in the service directory with the name `<service-name>.feature`.
```

Reviewer: It looks as if we have diverged on the naming convention between the `.md` and `.feature` files. 

What do you think of using a unique name for each taxonomy file, similar to the gherkin? Alternatively it might be the other way around, with generic names for the feature files within a service directory.

> - All higher-level groupings in the hierarchy should be completed with `taxonomy.md` and `<service-name>.feature` files containing the common features.

Reviewer: ```suggestion
- All higher-level groupings in the hierarchy have been completed with `taxonomy.md` and `<service-name>.feature` files containing the common features.
```

Reviewer: It looks as if we have diverged on the naming convention between the .md and .feature files.

What do you think of using a unique name for each taxonomy file, similar to the gherkin?

Alternatively it might be the other way around, with generic names for the feature files within a service directory.

Author: <service-name-short>-taxonomy
will be considered as the file name for bothe md and gherkin files.

E.g.: rdms-taxonomy.md & rdms-taxonomy.feature

> - All higher-level groupings in the hierarchy should be completed with `taxonmy.md` and `<service-name>.feature` files of its own containing the common features.

Reviewer 2: ```suggestion
- All higher-level groupings in the hierarchy should be completed with `taxonomy.md` and `<service-name>.feature` files containing the common features.
```

> - All higher-level groupings in the hierarchy should be completed with `taxonmy.md` and `<service-name>.feature` files of its own containing the common features.

Reviewer: ```suggestion
- All higher-level groupings in the hierarchy have been completed with `taxonmy.md` and `<service-name>.feature` files of its own containing the common features.
```

Reviewer: It looks as if we have diverged on the naming convention between the .md and .feature files.

What do you think of using a unique name for each taxonomy file, similar to the gherkin?

Alternatively it might be the other way around, with generic names for the feature files within a service directory.
